tests/utils/constants/constants.go

```diff
@@ -101,6 +101,11 @@ const (
 	PreferNoSchedule = "PreferNoSchedule"
 )
 
+// Machine pool
+const (
+	DefaultMachinePoolName = "worker"
```

Comment:
Where is that used ?

Reply:
Classic default worker pool related cases. @radtriste